src/form/Toggle/ToggleTheme.ts

```diff
@@ -18,16 +21,20 @@ export interface ToggleTheme {
 }
 
 const baseTheme: ToggleTheme = {
-  base: 'flex items-center justify-start cursor-pointer bg-surface box-border border border-panel-accent rounded-full',
-  disabled: 'cursor-not-allowed opacity-60',
+  base: 'flex items-center justify-start cursor-pointer bg-surface box-border border border-panel-accent rounded-full hover:bg-primary-hover transition-[background-color] ease-in-out duration-300',
+  disabled: 'cursor-not-allowed bg-transparent hover:bg-transparent',
   checked: 'justify-end bg-primary',
+  disabledAndChecked:
```

Comment:
These colors should be the palettes not specific instances

Reply:
looks like these gray's could be replaced by `bg-secondary-inactive`, or maybe we should adjust the primary-inactive colors a bit. That might be what austin means

Reply:
Updated